FullStackMVCAPP/FullStackMVCAPP/DataContext/GOTContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Data.Entity;
+
+namespace FullStackMVCAPP.DataContext
+{
+public class GOTContext : DbContext 
+{
+
+    public DbSet<FullStackMVCAPP.Models.House> Houses { get; set; }
+    public DbSet<FullStackMVCAPP.Models.Castle> Castles { get; set; }
+    public DbSet<FullStackMVCAPP.Models.Character> Characters { get; set; }
```

Comment:
:thumbsup: Good move to split the context file out into its own file.
Just be a little bit careful about indendation here mainly as this could become difficult to read and identify where the brackets are placed.
Additionally a good tip, the `FullStackMVCAPP.Models.House` can be simplified by importing a reference to the model using the `using` keyword